tests/contrib/hooks/test_gcp_sheets_hook.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""
+This test script tests GCPSheetsHook on a live Google sheet, to do so:
```

Comment:
Can you add some unit tests instead? The tests you have now requires some manual steps to setup external dependencies and they cannot be ran as part of the CI.

Reply:
will do, can you suggest some unit tests that would be helpful here?

Reply:
I would suggest mock out the google sheet client and verify the args passed to the client.